concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -0,0 +1,219 @@
+import mock
+
+from django.test            import override_settings
+from django.urls            import reverse
+from freezegun              import freeze_time
+from golem_messages         import message
+
+from core.models            import Message
+from core.models            import ReceiveStatus
+from core.tests.utils       import ConcentIntegrationTestCase
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+def get_provider_account_status_true_mock(_):
```

Comment:
As I have found out, the convention is that we want to name the parameters like this, starting their name with `_`.